nucypher/blockchain/eth/policies.py

```diff
@@ -175,23 +158,27 @@ def make_arrangements(self, network_middleware: RestMiddleware,
         selected_addresses = set()
         try:  # Sample by reading from the Blockchain
             actual_sample_quantity = math.ceil(target_sample_quantity * ADDITIONAL_URSULAS)
```

Comment:
Sampling `1.5 * len(kfrags)` Ursulas here in case some reject. Additionally sampling from the blockchain has another buffer of 1.7.  This makes the minimum number of nodes the network `(1.5 * N) * 1.7`

Reply:
Hmmm... Right.  OK, so are you asking anything here?  Or just commenting on the existing code?

Reply:
:heavy_check_mark: